### PR TITLE
Tidy up sanity checks (2021/Batch 3 - who's counting)

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Fixed a batch of sanity checks and removed lines from the ignore files

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
plugins/modules/remote_management/lxca/lxca_nodes.py
plugins/modules/remote_management/lxca/lxca_cmms.py
plugins/modules/cloud/webfaction/webfaction_site.py
plugins/modules/cloud/webfaction/webfaction_mailbox.py
plugins/modules/cloud/webfaction/webfaction_domain.py
plugins/modules/cloud/webfaction/webfaction_db.py
plugins/modules/cloud/webfaction/webfaction_app.py
plugins/modules/cloud/oneandone/oneandone_firewall_policy.py
plugins/modules/cloud/oneandone/oneandone_load_balancer.py
plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py
plugins/modules/cloud/oneandone/oneandone_private_network.py
plugins/modules/cloud/oneandone/oneandone_server.py
plugins/modules/cloud/packet/packet_sshkey.py
plugins/modules/cloud/profitbricks/profitbricks_volume.py
plugins/modules/cloud/profitbricks/profitbricks.py
plugins/modules/clustering/consul/consul.py
plugins/modules/clustering/consul/consul_kv.py
plugins/modules/clustering/consul/consul_session.py
plugins/modules/clustering/etcd3.py
plugins/modules/clustering/znode.py
plugins/modules/monitoring/datadog/datadog_event.py
plugins/modules/monitoring/datadog/datadog_monitor.py
plugins/modules/monitoring/icinga2_host.py
plugins/modules/monitoring/sensu/sensu_client.py
plugins/modules/monitoring/sensu/sensu_check.py
plugins/modules/monitoring/sensu/sensu_handler.py
plugins/modules/notification/pushbullet.py
plugins/modules/packaging/language/bundler.py
plugins/modules/packaging/language/maven_artifact.py
plugins/modules/packaging/language/pip_package_info.py

### CODE REVIEW

**Comment by russoz** [comment]
Changelog frag to be added after initial reviews, in case we have to rollback some of these changes.

**Comment by russoz** [comment]
ok, still got failing tests, will continue tomorrow...

**Comment by felixfontein** [comment]
Please remember that every change that adds `elements` in the argspec requires a changelog fragment. Also note that reviewing this will require quite a long time since there are so many changes that need a thorough investigation.

**Comment by Andersson007** [comment]
@russoz thanks for doing this!

folks, feel free to mark things as checked below (better to check at list twice though):

- [ ] plugins/modules/remote_management/lxca/lxca_nodes.py
- [ ] plugins/modules/remote_management/lxca/lxca_cmms.py
- [ ] plugins/modules/cloud/webfaction/webfaction_site.py
- [ ] plugins/modules/cloud/webfaction/webfaction_mailbox.py
- [ ] plugins/modules/cloud/webfaction/webfaction_domain.py
- [ ] plugins/modules/cloud/webfaction/webfaction_db.py
- [ ] plugins/modules/cloud/webfaction/webfaction_app.py
- [ ] plugins/modules/cloud/oneandone/oneandone_firewall_policy.py
- [ ] plugins/modules/cloud/oneandone/oneandone_load_balancer.py
- [ ] plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py
- [ ] plugins/modules/cloud/oneandone/oneandone_private_network.py
- [ ] plugins/modules/cloud/oneandone/oneandone_server.py
- [ ] plugins/modules/cloud/packet/packet_sshkey.py
- [ ] plugins/modules/cloud/profitbricks/profitbricks_volume.py
- [ ] plugins/modules/cloud/profitbricks/profitbricks.py
- [ ] plugins/modules/clustering/consul/consul.py
- [ ] plugins/modules/clustering/consul/consul_kv.py
- [ ] plugins/modules/clustering/consul/consul_session.py
- [ ] plugins/modules/clustering/etcd3.py
- [ ] plugins/modules/clustering/znode.py
- [ ] plugins/modules/monitoring/datadog/datadog_event.py
- [ ] plugins/modules/monitoring/datadog/datadog_monitor.py
- [ ] plugins/modules/monitoring/icinga2_host.py
- [ ] plugins/modules/monitoring/sensu/sensu_client.py
- [ ] plugins/modules/monitoring/sensu/sensu_check.py
- [ ] plugins/modules/monitoring/sensu/sensu_handler.py
- [ ] plugins/modules/notification/pushbullet.py
- [ ] plugins/modules/packaging/language/bundler.py
- [ ] plugins/modules/packaging/language/maven_artifact.py
- [ ] plugins/modules/packaging/language/pip_package_info.py

**Comment by Andersson007** [comment]
@russoz thanks for doing this!

folks, feel free to mark things as checked below (better to check at list twice though):

- [ ] plugins/modules/remote_management/lxca/lxca_nodes.py
- [ ] plugins/modules/remote_management/lxca/lxca_cmms.py
- [ ] plugins/modules/cloud/webfaction/webfaction_site.py
- [ ] plugins/modules/cloud/webfaction/webfaction_mailbox.py
- [ ] plugins/modules/cloud/webfaction/webfaction_domain.py
- [ ] plugins/modules/cloud/webfaction/webfaction_db.py
- [ ] plugins/modules/cloud/webfaction/webfaction_app.py
- [ ] plugins/modules/cloud/oneandone/oneandone_firewall_policy.py
- [ ] plugins/modules/cloud/oneandone/oneandone_load_balancer.py
- [ ] plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py
- [ ] plugins/modules/cloud/oneandone/oneandone_private_network.py
- [ ] plugins/modules/cloud/oneandone/oneandone_server.py
- [ ] plugins/modules/cloud/packet/packet_sshkey.py
- [ ] plugins/modules/cloud/profitbricks/profitbricks_volume.py
- [ ] plugins/modules/cloud/profitbricks/profitbricks.py
- [ ] plugins/modules/clustering/consul/consul.py
- [ ] plugins/modules/clustering/consul/consul_kv.py
- [ ] plugins/modules/clustering/consul/consul_session.py
- [ ] plugins/modules/clustering/etcd3.py
- [ ] plugins/modules/clustering/znode.py
- [ ] plugins/modules/monitoring/datadog/datadog_event.py
- [ ] plugins/modules/monitoring/datadog/datadog_monitor.py
- [ ] plugins/modules/monitoring/icinga2_host.py
- [ ] plugins/modules/monitoring/sensu/sensu_client.py
- [ ] plugins/modules/monitoring/sensu/sensu_check.py
- [ ] plugins/modules/monitoring/sensu/sensu_handler.py
- [ ] plugins/modules/notification/pushbullet.py
- [ ] plugins/modules/packaging/language/bundler.py
- [ ] plugins/modules/packaging/language/maven_artifact.py
- [ ] plugins/modules/packaging/language/pip_package_info.py

**Comment by Andersson007** [comment]
@russoz thanks for doing this!

folks, feel free to mark things as checked below (better to check at list twice though):

- [ ] plugins/modules/remote_management/lxca/lxca_nodes.py
- [ ] plugins/modules/remote_management/lxca/lxca_cmms.py
- [ ] plugins/modules/cloud/webfaction/webfaction_site.py
- [ ] plugins/modules/cloud/webfaction/webfaction_mailbox.py
- [ ] plugins/modules/cloud/webfaction/webfaction_domain.py
- [ ] plugins/modules/cloud/webfaction/webfaction_db.py
- [ ] plugins/modules/cloud/webfaction/webfaction_app.py
- [ ] plugins/modules/cloud/oneandone/oneandone_firewall_policy.py
- [ ] plugins/modules/cloud/oneandone/oneandone_load_balancer.py
- [ ] plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py
- [ ] plugins/modules/cloud/oneandone/oneandone_private_network.py
- [ ] plugins/modules/cloud/oneandone/oneandone_server.py
- [ ] plugins/modules/cloud/packet/packet_sshkey.py
- [ ] plugins/modules/cloud/profitbricks/profitbricks_volume.py
- [ ] plugins/modules/cloud/profitbricks/profitbricks.py
- [ ] plugins/modules/clustering/consul/consul.py
- [ ] plugins/modules/clustering/consul/consul_kv.py
- [ ] plugins/modules/clustering/consul/consul_session.py
- [ ] plugins/modules/clustering/etcd3.py
- [ ] plugins/modules/clustering/znode.py
- [ ] plugins/modules/monitoring/datadog/datadog_event.py
- [ ] plugins/modules/monitoring/datadog/datadog_monitor.py
- [ ] plugins/modules/monitoring/icinga2_host.py
- [ ] plugins/modules/monitoring/sensu/sensu_client.py
- [ ] plugins/modules/monitoring/sensu/sensu_check.py
- [ ] plugins/modules/monitoring/sensu/sensu_handler.py
- [ ] plugins/modules/notification/pushbullet.py
- [ ] plugins/modules/packaging/language/bundler.py
- [ ] plugins/modules/packaging/language/maven_artifact.py
- [ ] plugins/modules/packaging/language/pip_package_info.py

**Comment by Andersson007** [comment]
@russoz thanks for doing this!

folks, feel free to mark things as checked below (better to check at list twice though):

- [ ] plugins/modules/remote_management/lxca/lxca_nodes.py
- [ ] plugins/modules/remote_management/lxca/lxca_cmms.py
- [ ] plugins/modules/cloud/webfaction/webfaction_site.py
- [ ] plugins/modules/cloud/webfaction/webfaction_mailbox.py
- [ ] plugins/modules/cloud/webfaction/webfaction_domain.py
- [ ] plugins/modules/cloud/webfaction/webfaction_db.py
- [ ] plugins/modules/cloud/webfaction/webfaction_app.py
- [ ] plugins/modules/cloud/oneandone/oneandone_firewall_policy.py
- [ ] plugins/modules/cloud/oneandone/oneandone_load_balancer.py
- [ ] plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py
- [ ] plugins/modules/cloud/oneandone/oneandone_private_network.py
- [ ] plugins/modules/cloud/oneandone/oneandone_server.py
- [ ] plugins/modules/cloud/packet/packet_sshkey.py
- [ ] plugins/modules/cloud/profitbricks/profitbricks_volume.py
- [ ] plugins/modules/cloud/profitbricks/profitbricks.py
- [ ] plugins/modules/clustering/consul/consul.py
- [ ] plugins/modules/clustering/consul/consul_kv.py
- [ ] plugins/modules/clustering/consul/consul_session.py
- [ ] plugins/modules/clustering/etcd3.py
- [ ] plugins/modules/clustering/znode.py
- [ ] plugins/modules/monitoring/datadog/datadog_event.py
- [ ] plugins/modules/monitoring/datadog/datadog_monitor.py
- [ ] plugins/modules/monitoring/icinga2_host.py
- [ ] plugins/modules/monitoring/sensu/sensu_client.py
- [ ] plugins/modules/monitoring/sensu/sensu_check.py
- [ ] plugins/modules/monitoring/sensu/sensu_handler.py
- [ ] plugins/modules/notification/pushbullet.py
- [ ] plugins/modules/packaging/language/bundler.py
- [ ] plugins/modules/packaging/language/maven_artifact.py
- [ ] plugins/modules/packaging/language/pip_package_info.py

**Comment by russoz** [comment]
@Andersson007 thanks for putting the checklist! It seems I cannot tick the boxes though :(
Some modules had only documentation changes (like adding `type: str`):
- plugins/modules/cloud/packet/packet_sshkey.py
- plugins/modules/cloud/webfaction/webfaction_app.py
- plugins/modules/cloud/webfaction/webfaction_db.py
- plugins/modules/cloud/webfaction/webfaction_mailbox.py
- plugins/modules/clustering/consul/consul_kv.py
- plugins/modules/clustering/etcd3.py
- plugins/modules/monitoring/icinga2_host.py
- plugins/modules/notification/pushbullet.py
- plugins/modules/remote_management/lxca/lxca_cmms.py
- plugins/modules/remote_management/lxca/lxca_nodes.py

Those modules should be more straightforward to review and tick on the list.

File: plugins/modules/remote_management/lxca/lxca_nodes.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
   uuid:
     description:
       uuid of device, this is string with length greater than 16.
+    type: str
```
Comment (Andersson007) [inline plugins/modules/remote_management/lxca/lxca_nodes.py:23]: all of them are not defined as strings in INPUT_ARG_SPEC, shouldn't we add `type='str'` to every option there?

Comment (felixfontein) [reply plugins/modules/remote_management/lxca/lxca_nodes.py:23]: `type='str'` is the default in the argument spec, so it's not really needed. I personally prefer it, but I also know other people prefer not to.

File: plugins/modules/remote_management/lxca/lxca_cmms.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
   uuid:
     description:
       uuid of device, this is string with length greater than 16.
+    type: str
```
Comment (Andersson007) [inline plugins/modules/remote_management/lxca/lxca_cmms.py:23]: same as for `lxca_nodes.py`, the options are not defined as strings in the `INPUT_ARG_SPEC ` dict. Though the tests are green
@felixfontein any ideas?

File: plugins/modules/cloud/webfaction/webfaction_domain.py [hunk 3]
```diff
@@ -87,8 +93,8 @@ def main():
     module = AnsibleModule(
         argument_spec=dict(
             name=dict(required=True),
-            state=dict(required=False, choices=['present', 'absent'], default='present'),
-            subdomains=dict(required=False, default=[], type='list'),
+            state=dict(choices=['present', 'absent'], default='present'),
```
Comment (Andersson007) [inline plugins/modules/cloud/webfaction/webfaction_domain.py:96]:
```suggestion
            state=dict(type='str', choices=['present', 'absent'], default='present'),
```

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 4]
```diff
@@ -197,7 +208,7 @@
 
 EXAMPLES = '''
 - name: Create a monitoring policy
-  oneandone_moitoring_policy:
+  oneandone_monitoring_policy:
```
Comment (Andersson007) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:211]:
```suggestion
  community.general.oneandone_monitoring_policy:
```

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 5]
```diff
@@ -258,13 +269,13 @@
     wait: true
 
 - name: Destroy a monitoring policy
-  oneandone_moitoring_policy:
+  oneandone_monitoring_policy:
```
Comment (Andersson007) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:272]:
```suggestion
  community.general.oneandone_monitoring_policy:
```

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 6]
```diff
@@ -258,13 +269,13 @@
...
     auth_token: oneandone_private_api_key
     state: absent
     name: ansible monitoring policy
 
 - name: Update a monitoring policy
-  oneandone_moitoring_policy:
+  oneandone_monitoring_policy:
```
Comment (Andersson007) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:278]:
```suggestion
  community.general.oneandone_monitoring_policy:
```

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 7]
```diff
@@ -315,7 +326,7 @@
     state: update
 
 - name: Add a port to a monitoring policy
-  oneandone_moitoring_policy:
+  oneandone_monitoring_policy:
```
Comment (Andersson007) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:329]:
```suggestion
  community.general.oneandone_monitoring_policy:
```

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 8]
```diff
@@ -328,7 +339,7 @@
     state: update
 
 - name: Update existing ports of a monitoring policy
-  oneandone_moitoring_policy:
+  oneandone_monitoring_policy:
```
Comment (Andersson007) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:342]:
```suggestion
  community.general.oneandone_monitoring_policy:
```

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 9]
```diff
@@ -348,15 +359,15 @@
     state: update
 
 - name: Remove a port from a monitoring policy
-  oneandone_moitoring_policy:
+  oneandone_monitoring_policy:
```
Comment (Andersson007) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:362]:
```suggestion
  community.general.oneandone_monitoring_policy:
```

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 10]
```diff
@@ -348,15 +359,15 @@
...
     auth_token: oneandone_private_api_key
     monitoring_policy: ansible monitoring policy updated
     remove_ports:
      - port_id
     state: update
 
 - name: Add a process to a monitoring policy
-  oneandone_moitoring_policy:
+  oneandone_monitoring_policy:
```
Comment (Andersson007) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:370]:
```suggestion
  community.general.oneandone_monitoring_policy:
```

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 11]
```diff
@@ -368,7 +379,7 @@
     state: update
 
 - name: Update existing processes of a monitoring policy
-  oneandone_moitoring_policy:
+  oneandone_monitoring_policy:
```
Comment (Andersson007) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:382]:
```suggestion
  community.general.oneandone_monitoring_policy:
```

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 12]
```diff
@@ -386,7 +397,7 @@
     state: update
 
 - name: Remove a process from a monitoring policy
-  oneandone_moitoring_policy:
+  oneandone_monitoring_policy:
```
Comment (Andersson007) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:400]:
```suggestion
  community.general.oneandone_monitoring_policy:
```

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 13]
```diff
@@ -395,7 +406,7 @@
     state: update
 
 - name: Add server to a monitoring policy
-  oneandone_moitoring_policy:
+  oneandone_monitoring_policy:
```
Comment (Andersson007) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:409]:
```suggestion
  community.general.oneandone_monitoring_policy:
```

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 14]
```diff
@@ -404,7 +415,7 @@
     state: update
 
 - name: Remove server from a monitoring policy
-  oneandone_moitoring_policy:
+  oneandone_monitoring_policy:
```
Comment (Andersson007) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:418]:
```suggestion
  community.general.oneandone_monitoring_policy:
```

File: plugins/modules/remote_management/lxca/lxca_nodes.py (hunk 1 above)
Comment (Andersson007) [reply plugins/modules/remote_management/lxca/lxca_nodes.py:23]: @felixfontein ok, thanks for the info, will know

File: plugins/modules/remote_management/lxca/lxca_cmms.py (hunk 2 above)
Comment (Andersson007) [reply plugins/modules/remote_management/lxca/lxca_cmms.py:23]: discussed

**Comment by Andersson007** [comment]
@russoz also remember to add a changelog fragment mentioned by @felixfontein

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 15]
```diff
@@ -957,17 +968,17 @@ def main():
...
+            thresholds=dict(type='list', elements="dict", default=[]),
+            ports=dict(type='list', elements="dict", default=[]),
+            processes=dict(type='list', elements="dict", default=[]),
+            add_ports=dict(type='list', elements="dict", default=[]),
+            update_ports=dict(type='list', elements="dict", default=[]),
+            remove_ports=dict(type='list', elements="str", default=[]),
+            add_processes=dict(type='list', elements="dict", default=[]),
+            update_processes=dict(type='list', elements="dict", default=[]),
+            remove_processes=dict(type='list', elements="str", default=[]),
+            add_servers=dict(type='list', elements="dict", default=[]),
```
Comment (felixfontein) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:980]:
```suggestion
            add_servers=dict(type='list', elements="str", default=[]),
```
I *think* this should be str: https://github.com/ansible-collections/community.general/blob/0de6330d5c6ede5aea6987910d00df934323283c/plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py#L412-L413

**Review by felixfontein — COMMENTED** [review]
I think everything else looks good.

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py (hunk 15 above)
Comment (Andersson007) [reply plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:980]: we should add @russoz as a mainainer of **all** the modules, so he will know about possible things introduced:D
(i'm joking)

Comment (russoz) [reply plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:980]: ahahahaha the same we do every night, Pinky: try and take over the world! :-)

yes, that one definitely seems like a `str` that escaped my attention

File: plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py [hunk 16]
```diff
@@ -133,41 +136,49 @@
...
     description:
       - Processes to add to the monitoring policy.
     type: list
+    elements: dict
     required: false
   add_servers:
     description:
       - Servers to add to the monitoring policy.
     type: list
+    elements: dict
```
Comment (russoz) [inline plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py:151]:
```suggestion
    elements: str
```

**Review by Andersson007 — APPROVED** [review]

File: changelogs/fragments/1885-sanity-check-fixes-batch3.yml [hunk 17]
```diff
@@ -0,0 +1,18 @@
...
+  - plugins/modules/clustering/consul/consul_session.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/monitoring/datadog/datadog_monitor.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/monitoring/sensu/sensu_check.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/monitoring/sensu/sensu_client.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/monitoring/sensu/sensu_handler.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
+  - plugins/modules/packaging/language/bundler.py - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
```
Comment (felixfontein) [inline changelogs/fragments/1885-sanity-check-fixes-batch3.yml:18]:
```suggestion
  - oneandone_firewall_policy - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - oneandone_load_balancer - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - oneandone_monitoring_policy - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - oneandone_private_network - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - oneandone_server - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - profitbricks - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - profitbricks_volume - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - webfaction_domain - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - webfaction_site - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - consul - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - consul_acl - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - consul_session - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - datadog_monitor - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - sensu_check - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - sensu_client - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - sensu_handler - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
  - bundler - elements of list parameters are now validated (https://github.com/ansible-collections/community.general/pull/1885).
```

Comment (Andersson007) [reply changelogs/fragments/1885-sanity-check-fixes-batch3.yml:18]: Perfect eye, felixfontein:) I forgot to look at the fragment at all:)

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for fixing this!
@Andersson007 thanks for reviewing!

**Comment by Andersson007** [comment]
thanks you folks!

**Comment by felixfontein** [comment]
Manual backport to stable-1 in #1911.

**Pull request merged**